packages/studio-be/package.json

```diff
@@ -9,7 +9,7 @@
     "start": "cd ./out && cross-env NODE_PATH=./ cross-env BP_MODULES_PATH=B:/bp18/modules node index.js",
     "build": "yarn clean && yarn tsc",
     "clean": "node ./node_modules/rimraf/bin.js ./out",
-    "watch": "yarn clean && yarn tsc --watch"
```

Comment:
Because it should be a single-purpose command. And everytime I want to use yarn watch, it's when I'm working in the code, it always fails because my server is still running, so I have to kill the server, type yarn watch, then start it again. Better dx